SimDataFormats/GeneratorProducts/src/hepmc_rootio.cc

```diff
@@ -0,0 +1,29 @@
+#include <HepMC/GenParticle.h>
```

Comment:
@Dr15Jones although there is nothing wrong with it, I wonder whether calling this source as the namespace might not be a bit misleading

Reply:
Naming suggestions are welcome.

Reply:
may be HepMC_IOhelper ? I have no strong feeling about it

Reply:
I think I prefer what is already there. That file does contain the only functions in that namespace.

Reply:
fine for me